source/getting-started/index.md

```diff
@@ -44,11 +45,6 @@ If you get an outdated version of npm, run `npm install -g npm`.
 
 On Mac and Linux, you can improve file watching performance by installing [Watchman](https://facebook.github.io/watchman/docs/install.html).
 
-### PhantomJS (optional)
-
-You can run your tests from the command line with PhantomJS, without the
-need for a browser to be open. Consult the [PhantomJS download instructions](http://phantomjs.org/download.html).
-
```

Comment:
I'm not sure how you handle the guides being updated as versions are released, but I think this is slated to be replaced in v2.15 ([relevant commit](https://github.com/ember-cli/ember-cli/commit/2001c2d93359c42b14dc8da9775948d04bd6e188#diff-f7e2d0d5944207cb98b51991f8da57b0)).